rubin_scheduler/scheduler/surveys/field_survey.py

```diff
@@ -89,6 +92,7 @@ def __init__(
         flush_pad=30.0,
         detailers=None,
         filter_change_time=None,
+        sort_band_order=True,
```

Comment:
I would set the default to False. That's super confusing to have the Survey object sorting observations into alphabetical band order by default. Normally I would say that if you want to have logic that reorders observations that should just be done with a Detailer.

Reply:
Yes -- I only had it set to True because that was the previous behavior .. which I have no idea why it was that way now (unless it was something about the way sequences used to be defined ages ago? I don't know .. can't imagine how you could get things OUT of order if you wanted them in order to be honest). 
I agree you'd normally do this with a detailer. 
I could remove the sorting entirely, which is  what I'm inclined to do.